src/hooks/useStepHook.ts

```diff
@@ -52,7 +52,23 @@ function useStepHook() {
     return activeStepNumber + 1
   }
 
-  return { getNextStep, isLastStep }
+  const hasPreviousStep = (displayPrevious: boolean) => {
+    if (isJourneyRevisit(journeyStatus)) {
```

Comment:
You can invert the condition, to avoid have if inside and if sand return early.
```suggestion
    if (!isJourneyRevisit(journeyStatus)) {
      return displayPrevious 
    }
    ...
```

Reply:
Good idea. Done